plutus-core/untyped-plutus-core/src/UntypedPlutusCore/Evaluation/Machine/Cek/Internal.hs

```diff
@@ -163,6 +167,12 @@ this can make a surprisingly large difference.
 -- 'Name' is not necessary but we leave it here for simplicity and debuggability.
 type NTerm uni fun = Term NamedDeBruijn uni fun
 
+forgetAnn :: forall ann uni fun. NTerm uni fun ann -> NTerm uni fun Any
+forgetAnn = unsafeCoerce
+
+rememberAnn :: forall ann uni fun. NTerm uni fun Any -> NTerm uni fun ann
+rememberAnn = unsafeCoerce
```

Comment:
Uhh, I'm really uncomfortable about that. Let's discuss it during the tech meeting.